index.html

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Mohamed Allam - Automation & Web Dev</title>
+    <link rel="icon" href="./assets/images/profile-picture.png">
+    <link rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.0/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css" />
+    <link
+        href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;1,100;1,200;1,300&display=swap"
+        rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Barlow:ital@1&family=Genos&family=Tomorrow:ital,wght@0,100;0,200;0,300;0,400;1,100;1,200;1,300&display=swap"
+        rel="stylesheet">
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.27.0/moment.min.js"></script>
+</head>
+
+<body>
+    <div id="page-container">
+        <div id="profile-section" class="portfolio-section">
+            <!-- <div id="toggle-button">
+                <label class="switch">
+                    <input type="checkbox">
+                    <span class="slider"></span>
+                </label>
+            </div> -->
+            <div id="profile-pic">
+                <img src="./assets/images/profile-picture.png" width="200px" height="200px">
+            </div>
+            <div>
+                <h1 class="neon-glow font-a ease-in">MOHAMED ALLAM_</h1>
+            </div>
+            <nav>
+                <ul>
+                    <li>
+                        <a href="#webdev-projects">Web Dev Projects</a>
+                    </li>
+                    <li>
+                        <a href="#googleapps-automations">Automation Projects</a>
+                    </li>
+                    <li>
+                        <a href="#technologies">Technologies</a>
+                    </li>
+                    <li>
+                        <a href="https://drive.google.com/file/d/18YLF-KeJrdaHmwGo7QgoGfOZOnhC93U1/view?usp=sharing"
+                            target="_blank">Resume</a>
```

Comment:
love the fact that your CV opens in a new window, allowing whoever is looking at your portfolio to easily return to the page when they have finished reading.